tfhe/benches/integer/bench.rs

```diff
@@ -810,9 +819,9 @@ define_server_key_bench_scalar_default_fn!(
     rng_func: default_scalar
 );
 define_server_key_bench_scalar_default_fn!(
-    method_name: unchecked_small_scalar_mul,
+    method_name: unchecked_scalar_mul_parallelized,
     display_name: mul,
-    rng_func: mul_scalar
+    rng_func: default_scalar
```

Comment:
Shouldn't we also add `unchecked_scalar_add` and unchecked_scalar_sub` with this new  random generator ?

Reply:
Well here I guess it sould be `rng_func: mul_scalar`but we could add unchecked_scalar_sub/add to the benches if they are not benched

Reply:
They are already in the benchmarks. I'm just saying that they don't use the `rng_func` in their macro call. Look some lines above.

Reply:
But you're totally right about using `rng_func: mul_scalar` for this multiplication.

Reply:
hum  I see that they are already using it 🤔

Reply:
You're right my bad, I was checking the file on my laptop on a branch that is not up to date.
So just change `default_scalar` to `mul_scalar` for this multiplication and then we're good to go.